pkg/apis/core/types.go

```diff
@@ -650,6 +650,13 @@ type ISCSIVolumeSource struct {
 	// <target portal>:<volume name> will be created for the connection.
 	// +optional
 	InitiatorName *string
+	// iSCSI Replacement Timeout field required to set the iSCSI TimeoutSetting to wait for session re-establishment
```

Comment:
Better "ReplacementTimeout controls how long the iSCSI layer should wait for a timed-out path/session to reestablish itself before failing any commands on it."